src/orquestra/quantum/circuits/_builtin_gates.py

```diff
@@ -74,6 +74,8 @@ def builtin_gate_by_name(name) -> GateRef:
 
 
 # --- misc ----
+# The Delay gate acts as an identity, however delays further circuit execution
+# on the given qubit by n-units of time.
```

Comment:
I would add something along the line of:

"Therefore this gate will evaluate to identity on the simulator, but it will have effect if used on the actual hardware"